### PR TITLE
Make node fail on invalid path to bonds/wallets files

### PR DESCRIPTION
**Pull request by itegulov**
## Overview
What title says and enables GenesisTest tests as they were apparently disabled

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-937

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala [hunk 1]
```diff
@@ -222,28 +222,6 @@ object Genesis {
     unsignedBlockProto(body, header, List.empty[Justification], shardId)
   }
 
-  //FIXME delay and simplify this
-  def toFile[F[_]: Applicative: Log](
```
Comment (ArturGajowy) [inline casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala]: Now that's the ultimate way to deal with this FIXME :heart:

File: casper/src/test/scala/coop/rchain/casper/genesis/GenesisTest.scala [hunk 2]
```diff
@@ -61,94 +61,102 @@ class GenesisTest extends FlatSpec with Matchers with BlockDagStorageFixture {
...
-        log: LogStub[Task],
-        time: LogicalTime[Task]
-    ) =>
-      for {
-        _ <- fromInputFiles()(runtimeManager, genesisPath, log, time)
-        _ = log.warns.find(_.contains("bonds")) should be(None)
-      } yield log.infos.count(_.contains("Created validator")) should be(numValidators)
-  }
+  "Genesis.fromInputFiles" should "generate random validators when no bonds file is given" in taskTest(
+    withGenResources {
```
Comment (ArturGajowy) [inline casper/src/test/scala/coop/rchain/casper/genesis/GenesisTest.scala]:
```
Enable GenesisTest tests

They were not being run previously as `withGenResources` just returns
Task[Unit] which was not being forced to run
```
OMG we need to stop using Unit :scream:

cc @lukasz-golebiewski you'll like this :D

Comment (itegulov) [reply casper/src/test/scala/coop/rchain/casper/genesis/GenesisTest.scala]: IKR, I was like "I changed a bunch of behaviour in `Genesis.scala`; why the hell none of the tests failed?"

**Pull request merged**